.rubocop.yml

```diff
@@ -80,3 +92,237 @@ Style/GuardClause:
 # Case equality is not intrinsically problematic.
 Style/CaseEquality:
   Enabled: false
+
+# New cops since Rubocop 1.0.
+# We have to explicitly opt-in for new cops to apply
+# before the next major release.
```

Comment:
This list is automatically generated when running Rubocop with `NewCops: pending`. I've enabled all pending cops since 1.0.

No new cops will be added unless we explicitly declare them.

One can set `NewCops: pending` again to see if new cops have been released, and possibly add them here.